src/base/castlerandom.pas

```diff
@@ -47,11 +47,12 @@   TCastleRandom = class(TObject)
     function RandomBoolean: boolean;
     { Randomly provides "-1", "0" or "1" with equal chances. }
     function RandomSign: longint;
-    { Returns a random number in 0 .. High(LongWord) range. }
+    { Returns a random number in 1 .. High(LongWord) range. }
```

Comment:
If Seed always <> 0, then 
- RandomInt64 never returns 0
- RandomBoolean is a bit unfair: 2^31 -1 changes for false, 2^31 chances for true :) :)

Both of these are extremely minor issues of course, not worth correcting. But the true output range (1...xxxx) should be mentioned at the documentation of RandomInt64, just like it is now mentioned for Random32bit.